docs/source/model_design/vivarium_overview/data_sources/index.rst

```diff
@@ -134,29 +134,99 @@ For reference, `the decomposition rules for GBD 2021 can be found here <https://
+
 Pulling GBD Data using *Vivarium Inputs*
 ----------------------------------------
 
+There are two main code bases within the Vivarium software framework that are especially useful for interacting with GBD data, including `gbd_mapping <https://vivarium.readthedocs.io/projects/gbd-mapping/en/latest/index.html>`_ and `vivarium_inputs.interface <https://vivarium.readthedocs.io/projects/vivarium-inputs/en/latest/index.html>`_.
+
+Both of these code bases translate ID numbers used in GBD to human-readable text by default.
+
+Overview of :code:`gbd_mapping`
++++++++++++++++++++++++++++++++
+
+:code:`gbd_mapping` provides a convienient way to access all of the metadata associated with a given GBD entity (ex: diarrheal diseases cause or child growth failure risk factor), but does not return any estimates assoicated with that entity (ex: prevalence or relative risks).
+
+Overview of :code:`vivarium_inputs.interface`
+++++++++++++++++++++++++++++++++++++++++++++++
+
+:code:`vivarium_inputs.interface` provides simplified functions to query GBD data and reformats the data to be compatible with the data structure required for building Vivarium Artifact objects. :code:`vivarium_inputs.interface` generally returns data for the most up-to-date *complete* GBD round/release and does not allow for user-specification of prior rounds/releases -- ask the software engineers if you have questions about which GBD round/release is active in :code:`vivarium_inputs.interface` at any given time. Additionally, if there is any doubt as to which GBD versioning is being returned by a given :code:`vivarium_inputs.interface` call, you can utilize :code:`get_raw_data`, which will return full data including GBD versioning IDs for a given call.
+
+`For documentation on Vivarium Inputs, click here <https://vivarium.readthedocs.io/projects/vivarium-inputs/en/latest/index.html>`_.
+
+Some important notes and considerations not included in the documentation above are listed below:
+
 .. todo::
+   
+   List default behavior of get_measures/other functions once the GBD 2021 update is finalized, including things like:
+
+   - Returning most recent available year - note potential exception with risk effects?
+   - Filtering of draws (reduction of 1,000 COD draws down to 500 that are present in COMO)?
+   - Returning all ages/sexes and filling NANs with zeros
+   - Version ID behavior with GBD 2021? 
+   - Anything else?
 
-   this section
+.. list-table:: Notable default behavior of get_measures
+   :header-rows: 1
+
+   *  - Measure
+      - Data returned
+      - Note
+   *  - :code:`'incidence'`
+      - GBD_incidence / (1 - GBD_prevalence)
+      - By default, get_measures automatically converts GBD's "population-level incidence rates" to "susceptible population incidence rates" using the GBD estimate of prevalence. Note that if a model is using an alternative vlaue for prevalence, this rescaling should be done separately using that prevalence value.
```

Comment:
```suggestion
      - By default, get_measures automatically converts GBD's "population-level incidence rates" to "susceptible population incidence rates" using the GBD estimate of prevalence. Note that if a model is using an alternative value for prevalence, this rescaling should be done separately using that prevalence value.
```

I'm not sure that the rescaling should _always_ be done with the model's prevalence. I think it depends on what data informed GBD. If GBD was informed by incidence studies that measured incidence among the susceptible population, we'd want to rescale with GBD's prevalence always, I think.

(I realize with the way DisMod works this probably isn't binary.)